README.md

```diff
@@ -19,6 +19,7 @@ To contribute:
 
 | Name                                                                                                                                                       | Location                                | Application Period    | Notes                                                                                          | Date Posted |
 | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------- | --------------------- | ---------------------------------------------------------------------------------------------- | ----------- |
+| [Bell](https://jobs.bell.ca/ca/en/job/BECACA414139EXTERNALENCA/2024-Internship-Program-AI-Data-Engineering) | Toronto, Montreal | Oct 31 | AI & Data Engineering | Unknown 
```

Comment:
Maybe put the date of PR instead of Unknown in `Date Posted`?

Reply:
I will do that. Thanks